### ISSUE TITLE
Agency vs. institution vs. repository

### ISSUE THREAD

**Issue by elizhobart** [rbms-bsc/DCRMR#136]
BSC reviewers noted inconsistency in DCRMR usage of "agency," "institution," and "repository," with a suggestion that we normalize usage in the text. We discussed this briefly at our April 20 editorial group meeting and couldn't come to a quick consensus, with different members each preferring different terms.

- "Repository" is used only a few times (e.g., I.01.12.1, "printed textual monographs receiving special treatment within a repository")
- "Institution" is used around a dozen times (e.g., 1.2.32.2: "Take the indexing capabilities of the institution’s local system into consideration"
- "Agency" is used most frequently, over 50 times (e.g., "language and script of the cataloging agency")

What do you all think?

**Comment by jess-grz** [comment]
I tend to prefer "agency," which may more broadly encompass the individuals and corporate bodies who catalog or create metadata descriptions. Not every cataloger works on behalf of an institution. Some examples of non-institutional cataloging "agencies" are small historical societies, antiquarian booksellers, and external vendors. 

"Repositories" may also be problematic. Not all agencies, institutions, etc. that catalog resources actually own or physically house the materials. Community archiving initiatives are good examples of this scenario, in which an institution digitizes and creates metadata descriptions for a collection owned by a small organization or community group, which is housed by the group rather than the institution. Another example is an external vendor, although an institution, etc. would presumably work with the vendor to develop cataloging guidelines.

For what it's worth, the LC-PCC PSs in the new Toolkit mention "cataloging agency" only and not "institution." The PSs mention "repository" only as it relates to recording Title of manifestation for manuscripts and creating access points that incorporate the repository name or designation for physical manuscripts.

**Comment by deborahjleslie** [comment]
As I said in the meeting, I have a preference for respository, but can agree with Jessica's solution if we always preference 'agency' with 'cataloging.

**Comment by mjmascaro** [comment]
While I said at the meeting that I preferred institution, after reading Jessica's comment, I am now leaning towards "agency."

**Comment by hildebrand-r** [comment]
I have a slight preference for agency, and cannot say it isn’t simply that I am used to it. At the same time, I note that many instances of “cataloging agency” occur with reference to decisions or policies that in most cases would not be made by anyone other than an institution or repository. But as both “institution” and “repository” are subsumed by “agency” I think it is fine, and probably preferable to tailoring the language for each situation. As usual, I think it is good to be in sync with the LC-PCC PSs when possible.

**Comment by lizoknj** [comment]
I think there is a distinction worth preserving between “cataloging agency” and the body that owns/has custody of the material, whether called “institution” or “repository”. “Cataloging agency” is appropriate when referring to the cataloging/descriptive activities conducted within the institution, i.e. in the context of cataloging policies or language and script of cataloging. “Institution”/“repository” is appropriate in the context of the institution as owner/custodian. In the instruction below, "institution" appears twice: 

1.27.31 Access points for item-specific titles
1.27.31.1 Optionally, provide access for item-specific titles, such as a binder’s title. Identify the item using a designation of the holding institution (e.g., a name, acronym, or code). Provide identification such as a designation of the item’s physical location (e.g., a shelfmark) or an indication of the item’s copy number (if the institution holds more than one exemplar of the manifestation) in a Note on item if considered important (see 9.4.3.4).

In both cases, “cataloging agency” (or “agency”) feels wrong as a substitute.

Re whether to choose “repository” or “institution”: Initially I said that manuscripts people might prefer “repository”,  but Michelle pointed out that “repository” has digital associations. When I went back and looked at DCRM(MSS), I found that “institution” was used  24 times, “repository” 11 times (with no apparent difference in meaning), and "agency" 15 times (almost always to refer to the “cataloging” (sometimes “describing”) agency). Of the two terms, I would prefer “institution” because it has broader applicability than "repository" . I tend to associate "repository" with special collections units; if they form just part of the institution, it would be odd to refer to the local system of the repository, as opposed to the institution.
